decentraland/unity-renderer
# feat: BiW - Disable voxels loading for BiW V1

## What this PR includes?

### Fixes #384
Voxel tool is not polished nor explained to the user and will be disabled for V1.

![image](https://user-images.githubusercontent.com/64659061/118524346-57926780-b73e-11eb-87db-8e9b8f6c0bab.png)

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/BiW-disable-voxels&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=100,100
2. Press 'K' to open the Builer In World editor.
3. Notice that no voxels have been loaded into the catalog.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
